apis/controller/v1alpha1/devworkspaceoperatorconfig_types.go

```diff
@@ -87,13 +97,9 @@ type WorkspaceConfig struct {
 	// StorageClassName defines an optional storageClass to use for persistent
 	// volume claims created to support DevWorkspaces
 	StorageClassName *string `json:"storageClassName,omitempty"`
-	// DefaultStorageSize defines an optional map of sizes of persistent volume claims for each storage
-	// class used by DevWorkspaces.
-	// The available keys to the map are: "common", "per-workspace".
-	// Note that the "async" storage class also uses the PVC size set for the "common" storage class.
-	// If not specified, the storage class size defaults are:
-	// 10Gi for "common" (and "async"), 3Gi for "per-workspace".
-	DefaultStorageSize map[string]resource.Quantity `json:"defaultStorageSize,omitempty"`
+	// DefaultStorageSize defines an optional struct with fields to specify the sizes of Persistent Volume Claims for storage
+	// classes used by DevWorkspaces.
+	DefaultStorageSize *StorageSizes `json:"defaultStorageSize,omitempty"`
```

Comment:
maybe this should be `DefaultStorageSize *corev1.StorageSizes` instead?

Reply:
You can't add stuff to the corev1 package, that's owned by kubebuilder (usually we use `corev1` as an alias for `"k8s.io/api/core/v1"` since otherwise the import is `v1` which collides with everything :)

E.g.
```go
import (
	corev1 "k8s.io/api/core/v1"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
)
```
without an alias, Go will use the last section of the import (`v1` in both cases) so we have to alias them to `corev1` and `metav1`

Reply:
Okay makes sense :) So in other words, just having it has `DefaultStorageSize *StorageSizes` is okay?